docs/Documentation/Scripting/Building-Blocks/Integration-List.md

```diff
@@ -109,39 +109,12 @@ If you have this plugin you can use it's NPCs. I highly recommend you installing
 it's NPCs are way more immersive. Having Citizens also allows you to use NPCKill objective and to have moving NPC's
 in addition to the normal Npc functionality.
 
-### Npc Integration
+!!! info
+    In addition Citizens integration supports all [BetonQuest Npc](../../Features/Npcs.md) features.
 
-Citizens supports all [BetonQuest Npc](../../Features/Npcs.md) Features.
+## Events
```

Comment:
You can not change the heading level, it is intendet to be a sub headding of Citizens. The same for the other edited headings like objectives